src/Components/Artwork/Details.tsx

```diff
@@ -410,6 +410,13 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
       href
       title
       date
+      collectorSignals {
```

Comment:
So adding a fetch (possibly slowing things down) are going to be added here - but without actually rendering or being used yet?

If so - I get wanting to see performance earlier rather than later - but this PR would need to be reverted if there was an issue (and in general, we shouldn't do this longer than truly necessary). Maybe this fragment could only be included based on an Unleash flag?

Reply:
(or is the feature flag at the MP level, where it'll just resolve `collectorSignals: null` when not enabled?)

Reply:
Yeah, calling the dataloaders is controlled by feature flags in [MP](https://github.com/artsy/metaphysics/blob/eaf5a5255ed0ecf3383479890dbe281b768d43ad/src/lib/loaders/collectorSignalsLoader.ts#L61). This wouldn't have significant performance impact in production, right?

I think it could be good to enable globally in staging to observe performance early.

Reply:
Yea - if these just resolve as `null` and no fetch occurs when the feature flag is not enabled - cool! That shouldn't add any overhead. And then that means we can always turn it off when we have enough data or there's an issue or whatever.

In terms of enabling the flag - might as well enable it on staging more globally - see how Integrity likes it and can monitor performance.